pkg/store/storepb/mock.go

```diff
@@ -0,0 +1,33 @@
+package storepb
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+)
+
+// StoreClientMock is a structure for mocking StoreClient interface
+type StoreClientMock struct {
```

Comment:
I don't think this is necessary. See my comment here: https://github.com/improbable-eng/thanos/pull/895#discussion_r264281383